### ISSUE TITLE
EyeDropper button on ColorInput missing discernible text for Accessibility purposes

### ISSUE THREAD

**Issue by angelo-loria** [mantinedev/mantine#4227]
### What package has an issue

@mantine/core

### Describe the bug

The EyeDropper on the ColorInput component is rendered without any label, title, etc., which gets flagged as a critical issue in Deque's Axe scanner. When using a screenreader, there's no description of what the button is.

You can see the issue on the docs page for ColorInput- https://mantine.dev/core/color-input/#eye-dropper

Here is a screenshot of the error output from [Deque's Chrome extension](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd):
![image](https://github.com/mantinedev/mantine/assets/24941027/92053737-10e0-4dbd-8e3f-6d31a57b9775)

### What version of @mantine/hooks page do you have in package.json?

6.0.9

### Do you know how to fix the issue

Yes

### Are you willing to participate in fixing this issue and create a pull request with the fix

Yes

### Possible fix

Add `aria-label` attribute to EyeDropper button (I think, I'm new to this)

**Comment by rtivital** [comment]
Fixed in 6.0.11
